packages/docx-io/src/lib/mammoth.js/browser-demo/index.html

```
@@ -0,0 +1,33 @@
+    <meta charset="utf-8"> 
+    <title>Mammoth</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+  </head>
+  <body>
+    <div class="banner">
+      <div class="container">
+        <h1><a href="/">Mammoth</a></h1>
+      </div>
+    </div>
+    <div class="container">
+      <input id="document" type="file" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)